### PR TITLE
creating method to change session's table default name

### PR DESCRIPTION
**Pull request by 0syntrax0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 ## Features
 
-* Automatic loading and saving of session data via middleware.
+* Automatic loading and saving of session data via middleware.****
```
Comment (gregwebs) [inline README.md:10]: These look like changes by the IDE. This one looks wrong. These formatting changes should be reverted.

File: pgxstore/pgxstore.go [hunk 2]
```diff
@@ -51,7 +66,8 @@ func (p *PostgresStore) Find(token string) (b []byte, exists bool, err error) {
 // given expiry time. If the session token already exists, then the data and expiry
 // time are updated.
 func (p *PostgresStore) Commit(token string, b []byte, expiry time.Time) error {
-	_, err := p.pool.Exec(context.Background(), "INSERT INTO sessions (token, data, expiry) VALUES ($1, $2, $3) ON CONFLICT (token) DO UPDATE SET data = EXCLUDED.data, expiry = EXCLUDED.expiry", token, b, expiry)
+	query := fmt.Sprintf("INSERT INTO %s (token, data, expiry) VALUES ($1, $2, $3) ON CONFLICT (token) DO UPDATE SET data = EXCLUDED.data, expiry = EXCLUDED.expiry", p.sessionsTableName)
```
Comment (gregwebs) [inline pgxstore/pgxstore.go:69]: Now that the table is a variable, it would improve security to quote the table name

Comment (0syntrax0) [reply pgxstore/pgxstore.go]: ooh you're right!

File: README.md (hunk 1 above)
Comment (0syntrax0) [reply README.md]: reverted

File: README.md [hunk 3]
```diff
@@ -16,19 +16,20 @@
...
-    - [Configuring Session Behavior](#configuring-session-behavior)
-    - [Working with Session Data](#working-with-session-data)
-    - [Loading and Saving Sessions](#loading-and-saving-sessions)
-    - [Configuring the Session Store](#configuring-the-session-store)
-    - [Using Custom Session Stores](#using-custom-session-stores)
-      - [Using Custom Session Stores (with context.Context)](#using-custom-session-stores-with-contextcontext)
-    - [Multiple Sessions per Request](#multiple-sessions-per-request)
-    - [Enumerate All Sessions](#enumerate-all-sessions)
-    - [Compatibility](#compatibility)
+	- [Features](#features)
```
Comment (gregwebs) [inline README.md:19]: Still lots of changes here. Probably happening on save in the IDE

File: README.md [hunk 4]
```diff
@@ -26,6 +26,7 @@
     - [Configuring the Session Store](#configuring-the-session-store)
     - [Using Custom Session Stores](#using-custom-session-stores)
       - [Using Custom Session Stores (with context.Context)](#using-custom-session-stores-with-contextcontext)
+    - [Preventing Session Fixation](#preventing-session-fixation)
```
Comment (0syntrax0) [inline README.md:29]: `[Preventing Session Fixation]`  was not in the original version, but it should be linked.
Perhaps someone forgot to link it.

File: README.md (hunk 3 above)
Comment (0syntrax0) [reply README.md]: They were, I had to edit the file on the web, my VScode wouldn't let me keep spaces.

**Review by gregwebs — APPROVED** [review]

**Pull request merged**